api/src/test/java/io/opencensus/trace/AnnotationTest.java

```diff
@@ -11,11 +11,14 @@
  * limitations under the License.
  */
 
-package io.opencensus.trace.base;
+package io.opencensus.trace;
 
 import static com.google.common.truth.Truth.assertThat;
 
 import com.google.common.testing.EqualsTester;
+import io.opencensus.trace.Annotation;
```

Comment:
How come these weren't needed before but are needed now that AnnotationTest is in the same package?